Rocket/Rocket.Core/Commands/RocketCommandManager.cs

```diff
@@ -80,9 +81,9 @@ private IRocketCommand GetCommand(IRocketCommand command)
 
         public IRocketCommand GetCommand(string command)
```

Comment:
Would it not be possible to return a bool and out a IRocketCommand, then you dont have to check for null and just see if its a true or false return.

Reply:
Not the scope of this PR.

Reply:
Ik i was just pointing it out.